main.cpp

```diff
@@ -1,79 +1,10 @@
-#include <iostream>
-#include <assert.h>
+#include "ColorCode.h"
+#include "TelcoColorCoderTest.h"
 
-namespace TelCoColorCoder
+int main() 
 {
-    enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
-    enum MinorColor {BLUE, ORANGE, GREEN, BROWN, SLATE};
+    TelCoColorCoder::PrintTelcoColorCode();
```

Comment:
Think of a way to test `PrintTelcoColorCode` with asserts

Reply:
New test code added to test PrintTelcoColorCode api using asserts